### PR TITLE
[INLONG-7593][Manager] Change the InlongGroup status when executing Sort and Queue listeners

### LINKED ISSUE #7593: [Bug][Manager] Unable to delete InlongGroup
### What happened

Unable to delete group.

### What you expected to happen

Successfully delete the group.

### How to reproduce

Configure the group, and then delete it.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request
- Fixes #7593 

### Motivation

Fix unable to delete group.

### Modifications

Fix unable to delete group.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/sink/SinkResourceListener.java [hunk 1]
```diff
@@ -67,7 +68,22 @@ public ListenerResult listen(WorkflowContext context) {
         GroupResourceProcessForm form = (GroupResourceProcessForm) context.getProcessForm();
         String groupId = form.getInlongGroupId();
         log.info("begin to create sink resources for groupId={}", groupId);
-        groupService.updateStatus(groupId, GroupStatus.CONFIG_ING.getCode(), context.getOperator());
+        GroupOperateType operateType = form.getGroupOperateType();
+        String operator = context.getOperator();
+        switch (operateType) {
+            case INIT:
+                groupService.updateStatus(groupId, GroupStatus.CONFIG_ING.getCode(), context.getOperator());
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/sink/SinkResourceListener.java:75]:
```suggestion
                groupService.updateStatus(groupId, GroupStatus.CONFIG_ING.getCode(), operator);
```

**Review by vernedeng — APPROVED** [review]
LGTM

**Review by gong — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/sink/SinkResourceListener.java [hunk 2]
```diff
@@ -67,7 +68,22 @@ public ListenerResult listen(WorkflowContext context) {
         GroupResourceProcessForm form = (GroupResourceProcessForm) context.getProcessForm();
         String groupId = form.getInlongGroupId();
         log.info("begin to create sink resources for groupId={}", groupId);
-        groupService.updateStatus(groupId, GroupStatus.CONFIG_ING.getCode(), context.getOperator());
+        GroupOperateType operateType = form.getGroupOperateType();
+        String operator = context.getOperator();
+        switch (operateType) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/sink/SinkResourceListener.java:73]: After updating the StreamSink, it should only change the status of InlongStream.

Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/sink/SinkResourceListener.java:73]: In addition, StreamSink does not have operations such as init, suspend and restart, so these operations should not be judged here.

Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/sink/SinkResourceListener.java]: done

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**